public/docs/ts/latest/cli-quickstart.jade

```diff
@@ -122,157 +110,263 @@ figure.image-display
+        Soon there will be more components to declare. 
+  tr
+    td <code>assets/*</code>
+    td
+      :marked
+        A folder where you can put images and anything else you need to be copied wholesale 
+        when you build your application.
+  tr
+    td <code>environments/*</code>
+    td
+      :marked
+        This folder contains one file for each of your destination environments,
+        each exporting simple configuration variables to use on your application.
+        The files will be replaced on-the-fly when you build your app.
+        You might use a different API endpoint for development than you do for production.
+        Or maybe different analytics tokens.
+        Maybe even some mock services.
+        Either way, the CLI has you covered.
+  tr
+    td <code>favicon.ico</code>
+    td
+      :marked
+        Every site wants to look good on the bookmark bar.
+        Get started with your very own Angular icon.
+  tr
+    td <code>index.html</code>
+    td
+      :marked
+        The main html page that is served when someone visits your site.
+        Most of the time you'll never need to edit it.
+        The CLI will automatically add all `js` and `css` files when building your app so you 
+        never need to add any `<script>` or `<link>` tags here manually.
+  tr
+    td <code>main.ts</code>
+    td
+      :marked
+        The main entry point for your app.
+        Compiles the application with the [JiT compiler](glossary.html#jit)
+        and [bootstraps](guide/appmodule.html#main "bootstrap the application") 
+        the application to run in the browser.
+        You can also use the [AoT compiler](glossary.html#ahead-of-time-aot-compilation)
+        without changing any code by passing on `--aot` to `ng build` or `ng serve`.
+  tr
+    td <code>polyfills.ts</code>
+    td
+      :marked
+        Different browsers have different levels of support of the web standards.
+        Polyfills help normalize those differences.
+        You should be pretty safe with `core-js` and `zone.js`, but be sure to check out
+        the [Browser Support guide](guide/browser-support.html) for more information.
+  tr
+    td <code>styles.css</code>
+    td
+      :marked
+        Your global styles go here. 
+        Most of the time you'll want to have local styles in your components for easier maintenance,
+        but styles that affect all of your app need to be in a central place.
+  tr
+    td <code>test.ts</code>
+    td
+      :marked
+        This is the main entry point for your unit tests.
+        It has some custom configuration that might be unfamiliar, but it's not something you'll
+        need to edit.
+  tr
+    td <code>tsconfig.json</code>
+    td
+      :marked
+        Configuration for the TypeScript compiler.
+
+
+block root-files
+  :marked
+    ### The root folder
+    Although your app lives in `src/`, your *project* lives in the root folder.
```

Comment:
Done, but adjusted for the structure (described files are in root next to `src`):
>>>
    The `src/` folder is just one of the items inside the project's root folder. 
    Other files help you build, test, maintain, document, and deploy the app. 
    These files go in the root folder next to `src/`.